examples/image_classification/ios/ImageClassifier/Helper/ImageClassifierHelper.swift

```diff
@@ -0,0 +1,116 @@
+  init(model: Model, maxResults: Int, scoreThreshold: Float, runningModel: RunningMode, delegate: ImageClassifierHelperDelegate?) {
+    super.init()
+    guard let modelPath = model.modelPath else { return }
+    let imageClassifierOptions = ImageClassifierOptions()
+    imageClassifierOptions.runningMode = runningModel
+    imageClassifierOptions.maxResults = maxResults
+    imageClassifierOptions.scoreThreshold = scoreThreshold
+    imageClassifierOptions.baseOptions.modelAssetPath = modelPath
+    imageClassifierOptions.imageClassifierLiveStreamDelegate = runningModel == .liveStream ? self : nil
+    imageClassifier = try? ImageClassifier(options: imageClassifierOptions)
+    self.delegate = delegate
+  }
+
+  func classify(image: UIImage) -> ResultBundle? {
+    guard let mpImage = try? MPImage(uiImage: image) else { return nil }
+    do {
+      let startDate = Date()
+      guard let result = try imageClassifier?.classify(image: mpImage) else { return nil }
+      let inferenceTime = Date().timeIntervalSince(startDate) * 1000
+      return ResultBundle(inferenceTime: inferenceTime, imageClassifierResults: [result])
+    } catch {
+      print(error)
+      return nil
+    }
+  }
+
+  func classifyVideoFile(url: URL, inferenceIntervalMs: Double) async -> ResultBundle? {
+    guard let imageClassifier = imageClassifier else { return nil }
+    let startDate = Date()
+    let asset:AVAsset = AVAsset(url: url)
+    let generator = AVAssetImageGenerator(asset:asset)
+    generator.requestedTimeToleranceBefore = CMTimeMake(value: 1, timescale: 25)
+    generator.requestedTimeToleranceAfter = CMTimeMake(value: 1, timescale: 25)
+    generator.appliesPreferredTrackTransform = true
+    guard let videoDutationMS = try? await asset.load(.duration).seconds * 1000 else { return nil }
+    let frameCount = Int(videoDutationMS / inferenceIntervalMs)
+    var imageClassifierResults: [ImageClassifierResult?] = []
+    for i in 0..<frameCount {
+      let timestampMs = Int(inferenceIntervalMs) * i // ms
+      let image:CGImage?
+      do {
+        let time = CMTime(seconds: Double(timestampMs) / 1000, preferredTimescale: 600)
+        try image = generator.copyCGImage(at: time, actualTime:nil)
+      } catch {
+        print(error)
+         return nil
+      }
+      guard let image = image else { return nil }
+      let uiImage = UIImage(cgImage:image)
+      let result = try? imageClassifier.classify(videoFrame: MPImage(uiImage: uiImage), timestampInMilliseconds: timestampMs)
+      imageClassifierResults.append(result)
+    }
+    let inferenceTime = Date().timeIntervalSince(startDate) / Double(frameCount) * 1000
+    return ResultBundle(inferenceTime: inferenceTime, imageClassifierResults: imageClassifierResults)
+  }
+
+  func classifyAsync(videoFrame: CVPixelBuffer, timeStamps: Int) {
+    guard let imageClassifier = imageClassifier,
+          let image = try? MPImage(pixelBuffer: videoFrame) else { return }
```

Comment:
@khanhlvg @st-tuanmai 
Initialize using [MPImage(sampleBuffer:orientation:)](https://github.com/google/mediapipe/blob/master/mediapipe/tasks/ios/vision/core/sources/MPPImage.h#L167)
orientation can be passed in based on current `UIDecive` orientation and AVCaptureVideoDataOutput orientation.